docs/documentation.md

```diff
@@ -17,6 +17,24 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+# RabbitMQ Documentation
+
+Welcome to RabbitMQ documentation. RabbitMQ is a reliable and mature messaging and streaming broker, which is easy to deploy on cloud environments, on-premises, and on your local machine. With support for many different messaging protocols, it can scale from providing point-to-point messaging all the way up to globally distributed highly-available event streaming with rock-solid reliability. RabbitMQ is currently used by millions worldwide. 
+
+## How to use this documentation
+
+Use the navigation on the left to browse through documentation for your release of RabbitMQ. This table of contents is structured so it can be used by our two main RabbitMQ personas: Developer and Administrator. 
+
+## User: Developer
+
+Each developer scenario is different depending on the use case. For example, lets take the task of opening a connection, this route differs a lot depending on the messaging protocol a developers uses. This is just one factor, there are many more. You might want to start with [Getting Started](/tutorials), which is a set of tutorials that will guide you on how to use RabbitMQ with AMQP. If you need more details, then go the **How to Use RabbitMQ** section.
```

Comment:
I would skip the first part which is kind of obvious to the reader IMHO.

The paragraph would start at the "You might want to (...)", pointing the reader to the tutorials if they are new to RabbitMQ. Then talk about the "How to use RabbitMQ" section as you do in the last sentence.